frontend/lib/app.tsx

```diff
@@ -30,6 +30,13 @@ import { areAnalyticsEnabled } from './analytics';
 import MoratoriumBanner from './moratorium-banner';
 
 
+const ROUTES_WITH_MORATORIUM_BANNER = [
+  "/loc/splash",
+  "/hp/splash",
+  "/rh/splash",
+  "/"
+];
```

Comment:
This is cool for now since I doubt we'll internationalize the app before this moratorium is over, but just you know, if internationalization is enabled (which it isn't, on demo or production) these routes actually won't exist, since the locale-prefixed versions will exist instead (e.g. `/en/loc/splash`).  One way to future-proof it, as well as to avoid typos, is to use the `Routes` object/module rather than a hard-coded string, but there are some caveats around that that we probably don't want to deal with right now.  Anyhow just wanted to let you know for the future, I don't think you have to change anything here and now!

Reply:
Ah ok good to know! Hopefully the pandemic comes to an end before we have Tenant Platform internationalized...